lib/SGN/Controller/AJAX/Pedigrees.pm

```diff
@@ -280,11 +281,12 @@ sub get_full_pedigree_GET {
     my $schema = $c->dbic_schema("Bio::Chado::Schema");
     my $mother_cvterm = $schema->resultset("Cv::Cvterm")->find({name  => "female_parent"})->cvterm_id();
```

Comment:
sorry but when fetching any cvterm, please use SGN::Model::Cvterm. you can see the available cvterms and cv in system_cvterms.txt

Reply:
I'm not sure why I did this think to change all of them, I'll get on that.